interface/python_binding/setup.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2023, Shukant Pal and Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+from setuptools import setup
+
+__name__    = 'pyktx'
+__version__ = os.environ['LIBKTX_VERSION']
```

Comment:
Is KTX_VERSION set in the environment by the CMake build or is it only in CI builds?

Reply:
CMake build.